src/client/api/restful/get_outputs_id.h

```diff
@@ -11,6 +11,46 @@
 #include "core/address.h"
 #include "core/types.h"
 
+#define INDEXER_OUTPUTS_API_PATH "/api/plugins/indexer/v1/outputs"
+#define INDEXER_ALIASES_API_PATH "/api/plugins/indexer/v1/aliases"
+#define INDEXER_NFT_API_PATH "/api/plugins/indexer/v1/nfts"
+#define INDEXER_FOUNDRY_API_PATH "/api/plugins/indexer/v1/foundries"
```

Comment:
these are used in `get_outputs_id.c` only, should not put in the header.

Reply:
Done.